chart/mqtt-connector/values.yaml

```diff
@@ -22,6 +22,14 @@ asyncInvoke: false # Invoke via NATS using the function's asynchronous route
 basic_auth: true
 gateway_url: http://gateway.openfaas:8080
 
+resources:
+  requests:
+    memory: "128Mi"
+    cpu: "100m"
+  limits:
```

Comment:
Please comment out the limits section for all of these, so we only enforce a request and leave the limit up to the user, if they want it on.

Reply:
I was testing, commited by accident. I wanted to do it this way:
```
resources: {}
  # requests:
  #   memory: "64Mi"
  #   cpu: "100m"
  # limits:
  #   memory: "256Mi"
  #   cpu: "100m"
```

Reply:
We kept the request values in the other chart so that there's at least something

Reply:
Okay, I'll adjust it but here few things to consider in the future:
- helm-chart should provide minimal configuration of the application
- defaults should be indeed default values or default values of certain types - "", false/true, 0, null, etc.

Reply:
fixed.

Reply:
Thanks